requirements.txt

```diff
@@ -0,0 +1,2 @@
+openshift>=0.6.2
+requests-oauthlib
```

Comment:
I'm not sure what `requests-oauthlib` is used for either—I have never had to install it in my usage of this collection, and it's not listed as a dependency in the readme. For now, only the openshift library is a requirement.

Reply:
Ah... apparently if you use OpenShift's auth setup with `k8s_auth` you have to install it: https://github.com/ansible-collections/community.kubernetes/blob/main/plugins/modules/k8s_auth.py#L75 🤦‍♂️

Reply:
https://github.com/ansible-collections/community.kubernetes/blob/a16ac42f3edaa4419dffa5c04be85b8b4aa98611/plugins/modules/k8s_auth.py#L75

You can chase down the use with `HAS_REQUESTS_OAUTH`, and it fails on initialization if it doesn't have it.

One of the big things I still want to do is scrape collections for `HAS_*` variables and use that to get a speculative completeness metric for what dependencies are installed.

Reply:
FWIW it'd be useful to add code comments next to the entries explaining what needs them.